challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;500&display=swap" rel="stylesheet">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Document</title>
+  <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            primary_color: "#FFFFFF",
+            secondary_color: "#458FF6",
+            top_gradient: "#5A98F2",
+            bottom_gradient: "#67C3F3",
+            background: "#FAF9F6",
+          },
+          lineHeight: {
+            12: "60px",
+          },
+          rotate: {
+            160: "160deg",
+          },
+          padding: {
+            11.5: "46.5px",
+            15: "60px",
+          },
+          fontFamily: {
+            sans: ['Mulish', 'sans-serif'],
+          },
+          width: {
+            65: "259px",
```

Comment:
`w-65` should be `260px`, if we need exactly 259 px we could use the class `w-[259px]` in this case instead of extending tailwind config